markdown-basics.md

```diff
@@ -0,0 +1,58 @@
+# Learn Basic Markdown Syntax
+
+To create documentation on the GitHub platform, you need to be familiar with the basic Markdown syntax.
+
+This page shows you few examples of using the Markdown syntax to:
+
+* Add an image to your doc
+* Add a code block in your doc
+* Add an external link in your doc
```

Comment:
Please put a period at the end of each list item (simply as a way to practice the process). On aio, this is also standard practice.

Reply:
Added period at the end of each list item and pushed the changes.